src/cluster/cluster.cc

```diff
@@ -821,26 +821,26 @@ Status Cluster::CanExecByMySelf(const redis::CommandAttributes *attributes, cons
     int cur_slot = GetSlotIdFromKey(cmd_tokens[i]);
     if (slot == -1) slot = cur_slot;
     if (slot != cur_slot) {
-      return {Status::RedisExecErr, "CROSSSLOT Attempted to access keys that don't hash to the same slot"};
+      return {redis::ErrorKind::CrossSlot, "Attempted to access keys that don't hash to the same slot"};
```

Comment:
Is it possible to have `using redis::ErrorKind;` to write less code in such places?

Reply:
@torwig Do you mean to use `use redis::ErrorKind` to avoid the `redis::ErrorKind` prefix?

Reply:
@git-hulk Yes, to avoid typing the prefix.

Reply:
OK, that makes sense.